src/radical/pilot/session.py

```diff
@@ -285,11 +285,15 @@ def close(self, cleanup=True, terminate=True, delete=None):
             logger.error("Session object already closed.")
             return
 
+        # set defaults
+        if cleanup   == None: cleanup   = True
+        if terminate == None: terminate = True
+
```

Comment:
FWIW, I can stop converting those things, no need to start another style discussion...